webapp/src/app/pages/components/knowledge-graph/custom-recommendation-option/custom-recommendation-option.component.css

```
@@ -0,0 +1,135 @@
+:host ::ng-deep #search_concept_category_3 .p-dropdown .p-dropdown-item {
+  font-size: 12px;
+} */
+
+
+:host ::ng-deep #search_concept_category_3 .p-multiSelect {
+  width: 20em;
+}
+
+
+
+:host ::ng-deep .raning_progress_bar .p-progressbar .p-progressbar-label {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: ng-deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)